cfme/infrastructure/pxe.py

```diff
@@ -742,6 +780,32 @@ def create(self, name, provision_type, cancel=False):
         main_view.flash.assert_success_message(msg)
         return system_image_type
 
+    def delete(self, cancel=False, *sys_objs):
+        """
+        This methods deletes the System Image Type using select option,
+         hence can be used for multiple delete.
+
+        Args:
+            cancel: This is the boolean argument required for handle_alert
+            sys_objs: It's System Image Types object
+        """
+        view = navigate_to(self, 'All')
+        rows = view.entities
+        for sys_obj in sys_objs:
+            for row in rows:
+                if row.name.text == sys_obj.name:
+                    row[0].check()
+
+        view.toolbar.configuration.item_select("Remove System Image Types", handle_alert=cancel)
+
+        if not cancel:
+            view.toolbar.configuration.item_select("Remove System Image Types",
+                                                   handle_alert=not cancel)
+            main_view = self.create_view(PXESystemImageTypesView)
+            for sys_obj in sys_objs:
+                msg = 'System Image Type "{}": Delete successful'.format(sys_obj.name)
+                main_view.flash.assert_success_message(msg)
```

Comment:
**REQUIRED**: same here - we don't want to get any failures in other tests if we flash messages gets changed.
add separate test for flash messages

Reply:
Same as above.